source/slang-core-module/slang-embedded-core-module-source.cpp

```diff
@@ -149,6 +150,12 @@ static const BaseTypeConversionInfo kBaseTypes[] = {
      UINT_MASK,
      kBaseTypeConversionKind_Unsigned,
      kBaseTypeConversionRank_IntPtr},
+
+    {"bfloat16",
```

Comment:
should this be `bfloat16_t` to stay consistent with other explicitly sized types like `int64_t`?